apps/researcher/src/messages/en/messages.json

```diff
@@ -15,6 +15,18 @@
     "title": "Search for heritage objects",
     "description": "Start by typing a keyword to search.\n\nAfter this initial text search, search facets will appear to refine yor search results."
   },
+  "Signup": {
```

Comment:
A couple remark about the translations keys used. 

1. The main key used by `getTranslations` is `PascalCase` So this will be "SignUp"
2. The keys uses for the translations are `camelCase`. So `title` is correct, the rest needs be start with a lower case.
3. The keys are now coupled with the html (h1, h2, p) and styling (`NoteBold`). I would prefer if the keys don't say anything about the markup or styling. I don't want to changes keys if the markup changes.

Bonus point. Instead of 3 keys `AddP1`, `NoteBold`, `NoteText` you could make it one:
```
{ 
   "addText": "One of the most important functionalities is that users can add there narratives to the objects on this site. Users can tell how the objects are being used or give it the name in own language.\n\n<important>Please note</important>: To add a narrative you need to create an account using Google, Facebook or LinkedIn."
}

You can you the translation in the page with:

```
    {t.rich('addText', {
      important: text => <em>{text}</em>,
    })}
```

With all these point together you will get this:


```
    "SignUp": {
      "pageTitle": "Create an account",
      "whyTitle": "Why create an account?",
      "whyText": "With an account you have access to more functionality of this website. You can add your narratives to the objects, be part of a community or create lists to organise objects.\n\nThe reason one needs to create an account is that we want to know who adds the data. This is one of the ways we prevent anonyous spamming.",
      "whatTitle": "What can I do with an account?",
      "addTitle": "Add your narratives to objects",
      "addText": "One of the most important functionalities is that users can add there narratives to the objects on this site. Users can tell how the objects are being used or give it the name in own language.\n\n<important>Please note</important>: To add a narrative you need to create an account using Google, Facebook or LinkedIn.",
      "useTitle": "Use communities",
      "useText": "Communities are groups of people that represent a specific culture or geographical area. Anyone with an account can become a member of a community. You can also create your own community."
    },
```